src/libANGLE/renderer/metal/DisplayMtl.mm

```diff
@@ -301,17 +302,19 @@ bool IsMetalDisplayAvailable()
     config.colorComponentType = EGL_COLOR_COMPONENT_TYPE_FIXED_EXT;
 
     constexpr int samplesSupported[] = {0, 4};
+    constexpr int sizesSupported[] = {8, 16};
 
+    for (int size : sizesSupported)
     for (int samples : samplesSupported)
     {
         config.samples       = samples;
         config.sampleBuffers = (samples == 0) ? 0 : 1;
 
         // Buffer sizes
-        config.redSize    = 8;
-        config.greenSize  = 8;
-        config.blueSize   = 8;
-        config.alphaSize  = 8;
+        config.redSize    = size;
```

Comment:
Based on your explanation I understand now this is not correct. Instead I should only advertise only one RGBA16 format, and set allowed depth/stencil to 0 for that format.

What should I use for `renderTargetFormat` and `depthStencilFormat` for this new config?